frex/utils/constraints/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .constraint_type import ConstraintType
```

Comment:
organization of these constraints? maybe move to 'models' instead of weirdly keeping in 'utils' like now?

Reply:
Models, yes. Maybe `models.constraints`.